lib/ansible/galaxy/token.py

```diff
@@ -35,6 +48,93 @@
 
 display = Display()
 
+AH_TOKEN_EXPIRES_N_MIN_EARLY = 1
+AH_AUTH_URL = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
+AH_JWKS_URI = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs"
```

Comment:
I'd expect these to be in the constants/config module, so it's possible for the users to override them cleanly.

Reply:
We don't configure AH by default, just document the auth_url `https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token`. If I set the AH JWKS in config, it would break non-AH servers using KeycloakToken.

I'd like to fix AH hanging on import without requiring people change anything, but I'm not sure how to do that without hardcoding. I wish `auth_url` had been an option for the OpenID configuration instead (https://sso.redhat.com/auth/realms/redhat-external/.well-known/openid-configuration), so the `token_endpoint` and `jkws_uri` could be retrieved from the same source. I wonder if that would be reasonable? (fix would still require people to change something, but wouldn't require 2 options to do the same task)